packages/core-plugin-api/src/apis/definitions/auth.ts

```diff
@@ -450,3 +470,14 @@ export const atlassianAuthApiRef: ApiRef<
 > = createApiRef({
   id: 'core.auth.atlassian',
 });
+
+/**
+ * Provides authentication towards Pinniped APIs.
+ *
+ * @public
+ * @remarks
+ */
+export const pinnipedSupervisorApiRef: ApiRef<PinnipedSupervisorApi> =
```

Comment:
Maybe nice for consistency with the other refs in this file
```suggestion
export const pinnipedSupervisorAuthApiRef: ApiRef<PinnipedSupervisorApi> =
```